server/render.js

```diff
@@ -55,7 +55,8 @@ export default (app) => {
           'Storing server data',
           'Feedback messages',
           'Popups',
-          'Factories'
+          'Factories',
+          'Long Running Actions'
```

Comment:
And maybe call it `long running services`? The action ends, it is the service that keeps things going :)